standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -466,25 +466,30 @@ public Configuration getConf() {
   @Override
   @RetrySemantics.ReadOnly
   public GetOpenTxnsInfoResponse getOpenTxnsInfo() throws MetaException {
-    return getOpenTxnsList(true).toOpenTxnsInfoResponse();
+    return getOpenTxnsList(true, null).toOpenTxnsInfoResponse();
```

Comment:
Please create a variation of getOpenTxnsList with just 1 argument that requires a new db connection. 
```
private OpenTxnList getOpenTxnsList(boolean infoFields){
return getOpenTxnsList(infoFields, getDbConn(Connection.TRANSACTION_READ_COMMITTED))
}
```